core/tests/test_utils.py

```diff
@@ -483,12 +483,54 @@ def get_expected_logout_url(self, slug):
         """Returns the expected logout URL."""
         return current_user_services.create_logout_url(slug)
 
+    def get_response(self, url, params=None, expect_errors=False,
```

Comment:
Couple of thoughts:
- Do we ever use "params"? If not, maybe drop it.
- Do we actually need expect_errors, given that we can infer it from expected_status_int? If not then consider dropping it from the arglist and computing it within this function in order to pass the correct value to self.testapp.get().

Reply:
OK, let's leave it then. Thanks!

Reply:
Btw might be helpful to add an args docstring to this (for exactly this reason) -- in it, you can explain usage of params.

Reply:
done